src/chapters/html/studio.rst

```diff
@@ -0,0 +1,60 @@
+Studio: Making Headlines
+========================
+
+Getting Ready: Developer Tools
+------------------------------
+
+As you've learned, debugging code is an essential part of coding. When it comes to working with web pages, browser developer tools are indispensable.
+
+In this course, we will mostly use Firefox as our browser. If you haven't installed Firefox, you can get it from `Mozilla <https://www.mozilla.org/en-US/firefox/new/>`_. Other browsers have good developer tools as well (Chrome, in particular) but we're going to stick with Firefox. If you are more comfortable with another browser, you'll find that most of the developer tools functionality exists in other browsers' dev tools as well.
+
+Firefox's developer tools contain *a lot* of functionality, and we will only begin to touch on it here, learning more of its functionality later in the unit. In this class, we'll learn about the tools available for working with HTML elements.
+
+Walkthrough
```

Comment:
None of the other studios have walkthroughs. Maybe move to instructor handbook?

Reply:
I rephrased the content here so that it's oriented toward the student. Since we'll be introducing these concepts in a chapter, there won't need to be a walkthrough.

Reply:
Sounds good!